### DISCUSSION TITLE
CNS类

### DISCUSSION THREAD

**Discussion by 17602081651** [community/community#73270] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

是否可以实现cns多用户管理，进而实现查看某一用户流量使用情况。

**Comment by queenofcorgis** [comment]
Hi @17602081651 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
